templates/documents/general_view.html

```diff
@@ -309,8 +309,42 @@
+            {elseif $file->get_mimetype() eq "application/dicom"}
+            <!-- DWV -->
+            <div id="dwv" src="{$web_path}as_file=false">
+                <!-- Toolbar -->
+                <div class="toolbar"></div>
+                <!-- Layer Container -->
+                <div class="layerContainer">
+                    <canvas class="imageLayer">Only for HTML5 compatible browsers...</canvas>
+                </div><!-- /layerContainer -->
+            </div><!-- /dwv -->
+            <!-- Third party (dwv) -->
+            <script type="text/javascript" src="{$assets_static_relative}/modernizr/dist/modernizr-build.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next/i18next.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-xhr-backend/i18nextXHRBackend.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-browser-languagedetector/i18nextBrowserLanguageDetector.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/konva/konva.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/magic-wand-js/js/magic-wand-min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/jszip/dist/jszip.min.js"></script>
+            <!-- Third party (viewer) -->
+            <!--<script type="text/javascript" src="{$assets_static_relative}/jquery-min-2-1-4/index.js"></script>-->
+            <script type="text/javascript" src="{$assets_static_relative}/jquery-ui-1-12-1/jquery-ui.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/flot/jquery.flot.js"></script>
+            <!-- decoders -->
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/decoders/pdfjs/jpx.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/decoders/pdfjs/util.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/decoders/pdfjs/arithmetic_decoder.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/decoders/pdfjs/jpg.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/decoders/rii-mango/lossless-min.js"></script>
+            <!-- Local (dwv) -->
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-min-0-21-0/dist/dwv.min.js"></script>
+            <!-- Custom (Work in Progress) -->
+            <script type="text/javascript" src="{$webroot}/library/js/dwv/i18n.js"></script>
+            <!-- Main -->
+            <script type="text/javascript" src="{$webroot}/library/js/dwv/appgui.js"></script>
+            <script type="text/javascript" src="{$webroot}/library/js/dwv/applauncher.js"></script>
```

Comment:
@kofiav , Where did the 3 scripts above come from (a standard package/script or custom script)?

Reply:
@bradymiller

I manually put all 3 scripts in that location. 

`appgui.js` and `applauncher.js` both contain the custom JS code to construct the viewer. I wanted to keep them in separate files and was unsure where to put them. What is the 'best practice' as far as OpenEMR is concerned?

`i18n.js` was a dependency that I put in `/library/js/dwv` temporarily to get everything to work. I think it can be currently removed and pulled in using bower.

Reply:
The first 2 are in a good place. Should get standard author/copyright headers at the top of them, though. See here for a standard header example (replace the copyright/author tags with yours):
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock

And agree that to remove 3rd one and use it from within the standard bower library(if it exists there).

Reply:
Great, makes sense. Thanks for the review 😄  

I'll make the required changes and do another push when I'm able.